core/main_test.go

```diff
@@ -33,7 +33,7 @@ func TestInitRepo(t *testing.T) {
 	if err := InitRepo(InitConfig{
 		Account:  accnt,
 		RepoPath: vars.repoPath,
-		ApiAddr:  fmt.Sprintf("127.0.0.1:%s", GetRandomPort()),
+		ApiAddr:  fmt.Sprintf("127.0.0.1:9999"),
```

Comment:
We going to do this throughout? Why the change?

Reply:
For some reason, the random ports were causing the api health test to fail sporadically